src/components/PuzzleSpace/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Layout, Card } from "antd";
+import "./index.css";
+
+const GameSpace = () => {
```

Comment:
Since there is single function in the component, you may rename GameSpace as PuzzleSpace.